pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -31,12 +31,6 @@ import (
 	_ "github.com/google/cadvisor/container/crio/install"
 	_ "github.com/google/cadvisor/container/systemd/install"
 
-	// Register cloud info providers.
```

Comment:
@utkarsh2102 please leave this section in for now.

Reply:
Re-added! 👍🏻